ocs_ci/ocs/acm/acm.py

```diff
@@ -41,19 +44,35 @@ def import_cluster_ui(self, cluster_name, kubeconfig_location):
 
         """
         self.navigate_clusters_page()
-        self.do_click(self.page_nav["Import_cluster"])
+        time.sleep(40)
+        self.driver.find_element_by_id("importCluster").click()
+        log.info("Clicked on Import cluster")
+        time.sleep(60)
         self.do_send_keys(
             self.page_nav["Import_cluster_enter_name"], text=f"{cluster_name}"
         )
+        log.info("Keys sent")
+        log.info("sent input to the text box")
         self.do_click(self.page_nav["Import_mode"])
         self.do_click(self.page_nav["choose_kubeconfig"])
         log.info(f"Coping Kubeconfig {kubeconfig_location}")
```

Comment:
*Copying

Reply:
hmm, probably from earlier code, I can change that as well